BSP/Simulator/DataGeneration/Fans.py

```diff
@@ -0,0 +1,27 @@
+import csv
+import os
+import fcntl
+import config
+
+#This module will hold the statuses of the Fans on the BPS.
+
+#path name to file
+file = config.directory_path + config.files['Fans']
+
+#returns integer representing duty cycle
+def read():
```

Comment:
Can you call this somewhere in simulate.py, so we can see the status of the fans when we are running the simulator?